public/lib/helpers/timeline_helper.js

```diff
@@ -75,6 +75,32 @@ define(function (require) {
     };
 
     /**
+     * pluckDate returns date field value/raw value
+     *
+     * @param hit the document of the event
+     * @param params configuration parameters for the event
```

Comment:
There is no params 
instead there is field and fieldSequence 
please update the description